### ISSUE TITLE
Debian 12.6 MultiPHP install error: unable to locate package php5.6*

### ISSUE THREAD

**Issue by Brain8torm** [myvesta/vesta#184]
Hi! I installed myVesta for the first time on VDS c Debian 12.6. I try to install Multi PHP and get such errors. I'm attaching a screenshot. How to solve this problem?
![screenshot-20240725-001](https://github.com/user-attachments/assets/b5a2fdbc-cc85-4189-8112-6f992a537264)

**Comment by myvesta** [comment]
I just checked, and everything went fine.
Run `apt update` and paste output here; maybe you have an old sury signing key.

**Comment by Brain8torm** [comment]
When trying to run apt update, it returns an error

```
Hit:1 http://mirror.hoztnode.net/debian-security bookworm-security InRelease
Hit:2 http://mirror.hoztnode.net/debian bookworm-updates InRelease
Hit:3 http://mirror.hoztnode.net/debian bookworm InRelease
Err:4 https://packages.sury.org/php bookworm InRelease    
  451   [IP: 93.189.63.149 443]
Hit:5 https://nginx.org/packages/mainline/debian bookworm InRelease
Hit:6 https://apt.myvestacp.com/bookworm bookworm InRelease
Reading package lists... Done
E: Failed to fetch https://packages.sury.org/php/dists/bookworm/InRelease  451   [IP: 93.189.63.149 443]
E: The repository 'https://packages.sury.org/php bookworm InRelease' is not signed.
N: Updating from such a repository can't be done securely, and is therefore disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration details.

```

**Comment by Brain8torm** [comment]
It seems the problem has been solved. Yes, indeed, at first it was clear that the error was in outdated keys. But in the end, the problem is that when I tried to go to the server URL with the update, I found that they were unavailable from my location. Therefore, I fixed this error by configuring a proxy for apt.
The apt update command was executed successfully and the Multi PHP installation was completed without errors. Thanks!
